vitest.config.ts

```diff
@@ -0,0 +1,61 @@
+import { defineWorkersConfig } from '@cloudflare/vitest-pool-workers/config';
+
+export default defineWorkersConfig({
+  test: {
+    poolOptions: {
+      workers: {
+        miniflare: {
```

Comment:
This is wrong, it should just reference the wrangler.jsonc, don’t need to config mini flare.

import { defineWorkersConfig } from "@cloudflare/vitest-pool-workers/config";

export default defineWorkersConfig({
  test: {
    poolOptions: {
      workers: {
        wrangler: { configPath: "./wrangler.jsonc" },
      },
    },
  },
});


Source: https://developers.cloudflare.com/workers/testing/vitest-integration/write-your-first-test/index.md